external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,588 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
+
+```json
+{ ...
+   "columns": {
+      "Number of Arthur's Bakes": {
+         "type": "observations",
+         "data_type": "integer",
+         "unit": {
+            "label": "Count"
+         },
+         "measure": {
+            "label": "Number of Stores"
+         }
+      }
+   }
+}
+```
+
+##### Pivoted multi-measure data set
+
+| Year | Location | Number of Arthur's Bakes | Revenue | Revenue Units  |
+|:-----|:---------|-------------------------:|--------:|:---------------|
+| 2022 | London   |                       35 |      25 | GBP (Sterling) |
+| 2021 | Cardiff  |                       26 |      18 | GBP (Sterling) |
+
+In this example, a second observation value column, `Revenue`,  has been added, with the associated unit contained in the `Revenue Units` column. As you can see, the `Revenue Units` column has been linked to the `Revenue` column through the `describes_observations` field. This must be formatted in exactly the same way for csvcubed to recognise the link and generate the correct results. However, there is no measure information associated for either of the observation value columns; this can be configured as follows:
+
+```json
+{ ...
+   "columns": {
+      "Number of Arthur's Bakes": {
+         "type": "observations",
+         "data_type": "integer",
+         "unit": {
+            "label": "Count"
+         },
+         "measure": {
+            "label": "Number of Stores"
+         }
+      },
+      "Revenue": {
+         "type": "observations",
+         "data_type": "decimal",
+         "measure": {
+            "label": "Revenue"
+         }
+      },
+      "Revenue Units": {
+         "type": "units",
+         "describes_observations": "Revenue"
+      }
+   }
+}
+```
+
+## Measures and Units (for the guide)
+
+Measures can either be attached to a Measure Column if there are multiple different measures in your data set, or to an Observation column's `measure` field if all observations in the cube have the same measure.
+Units can either be attached to a Unit Column if there are a mixture of units in your data set, or to an Observation column if all observations in the cube have the same unit.
+
+The definition of measures and units columns is similar in that it depends on the number of measures/units present in the data set. If only one measure/unit is present, then the measure/unit can be defined by being attached to the Observation column, by entering them into their own field in the Observation column's definition. When multiple measures/units are present in the data set, they can be defined in their own column so that the column may clearly show which of the possible measures is being counted, or which unit is being used to measure each value in observations.
```

Comment:
I think we need to link to the observations column examples of attaching measures and units  at this point else it gets a bit too abstract.